tests/tx/test_verification.py

```diff
@@ -0,0 +1,448 @@
+#  Copyright 2023 Hathor Labs
+#
+#  Licensed under the Apache License, Version 2.0 (the "License");
+#  you may not use this file except in compliance with the License.
+#  You may obtain a copy of the License at
+#
+#  http://www.apache.org/licenses/LICENSE-2.0
+#
+#  Unless required by applicable law or agreed to in writing, software
+#  distributed under the License is distributed on an "AS IS" BASIS,
+#  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#  See the License for the specific language governing permissions and
+#  limitations under the License.
+
+from unittest.mock import Mock, patch
+
+from hathor.crypto.util import get_address_from_public_key
+from hathor.manager import HathorManager
+from hathor.transaction import BitcoinAuxPow, Block, MergeMinedBlock, Transaction, TxInput, TxOutput
+from hathor.transaction.scripts import P2PKH
+from hathor.transaction.token_creation_tx import TokenCreationTransaction
+from tests import unittest
+from tests.utils import add_blocks_unlock_reward, create_tokens, get_genesis_key
+
+
+class BaseVerificationTest(unittest.TestCase):
```

Comment:
I just missed tests checking the `validate_full()` method. This method should call _all_ methods with the prefix `verify_`.

Reply:
Done in https://github.com/HathorNetwork/hathor-core/pull/824/commits/ee9826aae50c0419e2000f1e4c5b468e4f412ba0

Reply:
Thanks!